### DISCUSSION TITLE
Welcome to Gityzer Discussions!

### DISCUSSION THREAD

**Discussion by vansh-codes** [vansh-codes/Gityzer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by palchhinparihar** [comment]
hi, @vansh-codes. Palchhin here.

I checked the website and I am getting this error:
<img width="1895" height="964" alt="image" src="https://github.com/user-attachments/assets/3470bfd3-f5e4-4322-9db6-7df3d1114dcb" />

```bash
  documentation_url :  "https://docs.github.com/rest"
  message : "Bad credentials"
  status : "401"
```

**Comment by vansh-codes** [comment]
Thanks for reporting @palchhinparihar 
I've fixed the issue, please check now!
Do go through the website, do let me know what can you add 
Thanks!

**Comment by palchhinparihar** [comment]
@vansh-codes sure!
Just checked the site and have 2 things to fix. however, while raising the issue i saw that in the template you got `SWOC` but I am contributing under `WOCS` by Code Social. :)

**Reply by vansh-codes** [reply]
Updated @palchhinparihar !

**Reply by palchhinparihar** [reply]
Thanks.. just noticed that the labels you added ain't correct! 
For WOCS, the labels are `wocs` in lowercase and `level 1`, `level 2`, `level 3`, exactly like this as they have mentioned in the document and meeting.

**Reply by vansh-codes** [reply]
Done!

**Comment by palchhinparihar** [comment]
Hi, @vansh-codes. For the tagline generation you need to use `gemini-2.5-flash`, not `gemini-1.5-flash,` as the latter one is giving an error.
in the **/api/ai/route.ts**.

Or else this is the error we'll get:
```
  Error generating content: [Error: [GoogleGenerativeAI Error]: Error fetching from [https://generativelanguage.googleapis.com/v1beta/models/gemini-pro:generateContent](vscode-file://vscode-app/c:/Users/KAEPI/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html): [404 Not Found] models/gemini-pro is not found for API version v1beta, or is not supported for generateContent. Call ListModels to see the list of available models and their supported methods.] {
    status: 404,
    statusText: 'Not Found',
    errorDetails: undefined
  } 
```

Note: It also depends on the API key permissions.
